model/schema/variantDTO.yaml

```diff
@@ -0,0 +1,166 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/variantDTO
+name: variantDTO
+title: variantDTO
+
+imports:
+  - core
+  - variation
+  - allele
+  - linkml:types
+
+prefixes:
+  alliance: 'http://alliancegenome.org/'
+  linkml: 'https://w3id.org/linkml/'
+  gff: 'https://w3id.org/gff'
+  faldo: 'http://biohackathon.org/resource/faldo#'
+  biolink: 'https://w3id.org/biolink/vocab/'
+  NLMID: 'https://www.ncbi.nlm.nih.gov/nlmcatalog/?term='
+  schema: 'http://schema.org/'
+
+default_prefix: alliance
+default_range: string
+
+default_curi_maps:
+  - obo_context
+  - idot_context
+  - semweb_context
+  - monarch_context
+
+emit_prefixes:
+  - rdf
+  - rdfs
+  - xsd
+  - owl
+
+## ------------
+## DTO CLASSES
+## ------------
+classes:
+
+  VariantDTO:
+    is_a: GenomicEntityDTO
+    description: >-
+      Ingest class for an Allele object
+    slots:
+      - variant_type
+      - related_notes
+      - source_general_consequence
```

Comment:
`consequence` and `source_general_consequence` have domain `VariantLocation` (the mixin), but are here listed as slots for the `VariantDTO` (which does not have and probably should not have this mixin). I would've expected this to break things? Maybe the DTO objects are outside of some checks (like the `make` command that builds the big `json` and `yaml` files)?

Reply:
Looking at non-DTO variant file, consequence is only in the location associations. Should we remove consequence from the VariantDTO then, but leave it in various location association DTOs?